docker-compose.yml

```diff
@@ -57,9 +57,12 @@ services:
     profiles:
       - nondefault
     volumes:
-      - ./config/nginx/:/etc/nginx/
+      - ./config/nginx/nginx.conf:/etc/nginx/nginx.conf:ro
+      - ./config/nginx/nginxconfig.io/:/etc/nginx/nginxconfig.io/:ro
+      - ./config/nginx/sites-enabled/:/etc/nginx/sites-enabled/:ro
       - ./ssl-data/certbot/conf:/etc/letsencrypt
       - ./ssl-data/certbot/www:/var/www/_letsencrypt
+      - app-public:/var/www/david-runger-public:ro
```

Comment:
Here we link `public/` from the `web` service to `/var/www/david-runger-public` in the `nginx` service.